.gitignore

```diff
@@ -17,3 +17,6 @@ target
 # NPM utilities for building docs
 /node_modules/
 package-lock.json
+
+# Default sqlite db files
+envelope-buffer.*
```

Comment:
If we make the default path `.relay/envelope-buffer.sqlite3` or `.relay/db/envelope-buffer.sqlite3`, we don't need this entry. The db is also a runtime file so I think it would be a good place:

https://github.com/getsentry/relay/blob/7c4f618d44fe5a2bdedc6c779407871621344e94/.gitignore#L10-L11